data/futures/csvconfig/rollconfig.csv

```diff
@@ -1,21 +1,45 @@
 Instrument,HoldRollCycle,RollOffsetDays,CarryOffset,PricedRollCycle,ExpiryOffset
+30YRCONF,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,11
+30YRJUMBO,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,9
 AEX,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,19
 AEX_mini,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,19
 ALUMINIUM,FGHJKMNQUVXZ,-45,1,FGHJKMNQUVXZ,14
+ALUMINIUM_LME,FGHJKMNQUVXZ,-45,1,FGHJKMNQUVXZ,14
+AMERIBOR-1M,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,26
+AMERIBOR-3M,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,26
+AMERIBOR-T30,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,26
+AMX,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,14
+ASX,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,18
+ATX,HMUZ,-5,1,HMUZ,16
 AUD,HMUZ,-5,1,HMUZ,17
+AUD-ICE,HMUZ,-5,1,HMUZ,16
+AUD-SGX,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,12
+AUDCAD,HMUZ,-5,1,HMUZ,16
 AUDJPY,HMUZ,-5,1,HMUZ,17
+AUDJPY-SGX,FGHJKMNQUVXZ,-5,1,FGHJKMNQUVXZ,13
 AUD_micro,HMUZ,-5,1,HMUZ,17
-BBCOMM,HMUZ,-5,1,HMUZ,15
+AUSCASH,FGHJKMNQUVXZ,-20,1,FGHJKMNQUVXZ,30
+BARLEY,FHKNUX,-5,1,FHKNUX,14
 BB3M,HMUZ,-90,-1,HMUZ,15
+BBCOMM,HMUZ,-5,1,HMUZ,15
```

Comment:
Not sure why diff has flagged as seems to be unchanged, probably just extra spaces